src/rpcblockchain.cpp

```diff
@@ -367,8 +367,6 @@ UniValue gettxoutsetinfo(const UniValue& params, bool fHelp)
             + HelpExampleRpc("gettxoutsetinfo", "")
         );
 
-    LOCK(cs_main);
-
     UniValue ret(UniValue::VOBJ);
 
     CCoinsStats stats;
```

Comment:
`FlushStateToDisk()` already grabs the necessary locks: `LOCK2(cs_main, cs_LastBlockFile);`